app/controllers/worksheets_controller.rb

```diff
@@ -0,0 +1,36 @@
+class WorksheetsController < ApplicationController
+
+  helper_method :worksheet
+  attr_reader :worksheet
+
+  def new
+    @worksheet = current_user.worksheets.new
+  end
+
+  def create
+    @worksheet = current_user.worksheets.new(worksheet_params)
+    if worksheet.save
```

Comment:
wasn't sure where to redirect initially, working on redirecting to the corresponding topic path. Will have that in shortly